shared/src/main/scala/drt/shared/TimeUtil.scala

```diff
@@ -0,0 +1,11 @@
+package drt.shared
+
+import drt.shared.CrunchApi.MillisSinceEpoch
+
+object TimeUtil {
+  def millisToMinutes(millis: MillisSinceEpoch): Int = {
```

Comment:
One for another ticket - we should take advantage of `FiniteDuration` so we don't need to manually translate between millis and minutes etc. Something we should have done a long time ago and would have saved us from a lot of headaches!

Reply:
This means change the input type of WalkTime from Long to finiteDuration which sound like we trying to fix a lot other issues in one ticket . It is technical debt were are trying to fix here .